src/components/editor/html-editor.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div class="highlight form-group" @click.stop="$emit('inspect')">
+    <div class="draggable-handle">
+      <i class="fas fa-arrows-alt"></i>
+    </div>
+    <form-html-editor name="htmlEditor" @onFocus="$emit('inspect')" label="Html Editor" v-model="content" />
+  </div>
+</template>
+
+<script>
+import { FormHtmlEditor } from "@processmaker/vue-form-elements/src/components";
```

Comment:
I know this branch is just for testing, but elements should be imported from `@processmaker/vue-form-elements`. Reaching directly into `src/components` means we bypass the bundled components built with `npm run build-bundle`.